ui.tests/test-module/specs/recaptcha/recaptcha.runtime.spec.js

```diff
@@ -116,4 +119,45 @@ describe("Form Runtime with Recaptcha Input", () => {
             cy.get(`#${id}`).parent().should("not.have.class", bemBlock);
         })
     })
-})
\ No newline at end of file
+
+    function updateEnterpriseConfig(score) {
+        const secretKey = Cypress.env('RECAPTCHA_ENT_API_KEY');
+        cy.visit("/mnt/overlay/fd/af/cloudservices/recaptcha/properties.html?item=%2Fconf%2Fcore-components-it%2Fsamples%2Frecaptcha%2Fbasic%2Fsettings%2Fcloudconfigs%2Frecaptcha%2Fentscore").then(x => {
```

Comment:
This would fail with context path, please use the openPage API or the wrapper which we have created.

Reply:
Resolved by using openPage API